### PR TITLE
Feature/corporate rules

### PR DESCRIPTION
**Pull request by biancameira12**
#### What is the purpose of this pull request?

inStore's Corporate Fields requires specific rules and translated messages

#### What problem is this solving?

Adapt instore-profile for the needs of checkout-instore

#### How should this be manually tested?
Test Suites and on inStore app

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [X] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/rules/BRA.js [hunk 1]
```diff
@@ -123,50 +118,77 @@ export default {
...
-          .slice(0, 12)
-          .reduce(
-            (acc, cur, index) =>
-              acc + parseInt(cur, 10) * parseInt(firstWeights[index], 10),
-            0
-          )
+        // Validate DVs
+        let size = cnpj.length - 2
+        let numbers = cnpj.substring(0, size)
+        const digitos = cnpj.substring(size)
```
Comment (ArthurSampaio) [inline react/rules/BRA.js]: :eyes:

**Review by ArthurSampaio — CHANGES_REQUESTED** [review]
why you did rollback on cnpj validation?

File: react/rules/BRA.js (hunk 1 above)
Comment (biancameira12) [reply react/rules/BRA.js]: This is the validation that inStore currently uses. I tested the profile-form validation and it didn't work on some CNPJs I tested

Comment (ArthurSampaio) [reply react/rules/BRA.js]: `digitos`.....

Comment (biancameira12) [reply react/rules/BRA.js]: Good eyes

**Review by ArthurSampaio — APPROVED** [review]

**Pull request merged**